server.py

```diff
@@ -20,7 +20,7 @@
 CORS(app)
 # useful for testing
 # curl -i http://localhost:5050/api/v0/
```

Comment:
Does it serve at port 5050? If not, delete

Reply:
actually I'm going to change it back to the development server running on 5050 ok?